bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/async/TestRpcThrottler.java

```diff
@@ -100,12 +101,11 @@ public void run() {
       });
       // Give some time for the Runnable to be executed.
       Thread.sleep(10);
-      assertFalse(secondRequestRegistered.get());
+      assertEquals(1, secondRequestRegisteredLatch.getCount());
       resourceLimiter.markCanBeCompleted(id);
 
-      // Give more time for the Runnable to be executed.
-      Thread.sleep(10);
-      assertTrue(secondRequestRegistered.get());
+      // Now wait for the second request to be registered
+      secondRequestRegisteredLatch.await(1, TimeUnit.MINUTES);
```

Comment:
What do you think about doing assertTrue(secondRequestRegisteredLatch.await(1, TimeUnit.MINUTES))?